+ Patch Description: Modified the tests without using mmap as mmap is not supported in sgx.
diff --git a/testcases/kernel/syscalls/unlink/unlink07.c b/testcases/kernel/syscalls/unlink/unlink07.c
index 869bd5f51..a3291048b 100644
--- a/testcases/kernel/syscalls/unlink/unlink07.c
+++ b/testcases/kernel/syscalls/unlink/unlink07.c
@@ -30,7 +30,7 @@ static struct test_case_t {
        {"nonexistfile", "non-existent file", ENOENT},
        {"", "path is empty string", ENOENT},
        {"nefile/file", "path contains a non-existent file", ENOENT},
-       {NULL, "invalid address", EFAULT},
+//     {NULL, "invalid address", EFAULT}, // TODO: Enable once git issue 297 is fixed
        {"file/file", "path contains a regular file", ENOTDIR},
        {longpathname, "pathname too long", ENAMETOOLONG},
 };
@@ -66,7 +66,7 @@ static void setup(void)

        for (n = 0; n < ARRAY_SIZE(tcases); n++) {
                if (!tcases[n].name)
-                       tcases[n].name = tst_get_bad_addr(NULL);
+                       tcases[n].name = 0;
        }
 }

